<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Screen3Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>701</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/metro10.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Nova</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>TREND DATA VISUALIZATIONS</string>
    </property>
    <property name="indent">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SCATTERPLOTS</string>
    </property>
    <property name="indent">
     <number>13</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>EXCEL FILE</string>
    </property>
    <property name="indent">
     <number>13</number>
    </property>
   </widget>
   <widget class="QComboBox" name="DataVisualizeCombo1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxVisibleItems">
     <number>12</number>
    </property>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Weekends</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Weekdays</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Holidays</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Percentage Distribution</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="DataVisualizeCombo2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxVisibleItems">
     <number>12</number>
    </property>
    <item>
     <property name="text">
      <string>Sunday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thursday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Friday</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saturday</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Screen3_SubmitBtn1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Screen3_SubmitBtn_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
